mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -614,6 +614,16 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
             agentExecutionMetadata.index = messageIndex
         }
 
+        // If the user has uploaded an image, we need to move the (base64 encoded) image 
```

Comment:
small nitpick, but can you update this comment to explain the WHY instead of just the WHAT. In particular, I'm wondering why don't we just send the image in the additionalContext instead of the agentExeuctionMetadata?